share/wake/lib/system/job.wake

```diff
@@ -229,7 +229,19 @@ export def editPlanShare (f: Boolean => Boolean): Plan => Plan =
 export def getPlanHash (plan: Plan): Integer =
     def Plan _ cmd _ env dir stdin _ _ _ _ _ _ _ _ isAtty = plan
     def isAttyStr = if isAtty then "true" else "false"
-    def sig = cat (isAttyStr, "\0\0", implode cmd, "\0\0", implode env, "\0\0", dir, "\0\0", stdin,)
+
+    def sig =
```

Comment:
This one looks better like this I think.

Reply:
lol we pretty much agreed on each case. I'm working on cleaning these up now